forms-flow-data-analysis-api/requirements.txt

```diff
@@ -1,67 +1,111 @@
-Flask-Migrate==4.0.5
+Brotli==1.1.0
+Deprecated==1.2.18
+Flask-Caching==2.3.1
+Flask-Migrate==4.1.0
+Flask-Moment==1.0.6
 Flask-SQLAlchemy==3.1.1
-Flask==2.3.3
-Jinja2==3.1.5
-Mako==1.2.4
-MarkupSafe==2.1.3
-PyYAML==6.0.1
-SQLAlchemy-Utils==0.41.1
-SQLAlchemy==2.0.21
-Werkzeug==3.0.6
-alembic==1.12.0
-aniso8601==9.0.1
-attrs==23.1.0
-blinker==1.6.2
-blis==0.7.10
-cachelib==0.10.2
-catalogue==1.0.2
-certifi==2024.7.4
-charset-normalizer==3.2.0
-click==8.1.7
-cymem==2.0.8
-ecdsa==0.18.0
-filelock==3.12.4
-flask-jwt-oidc==0.3.0
-flask-restx==1.1.0
-fsspec==2023.9.1
-gunicorn==22.0.0
-huggingface-hub==0.26.5
-idna==3.7
-itsdangerous==2.1.2
-joblib==1.3.2
-jsonschema-specifications==2023.7.1
-jsonschema==4.19.1
+Flask==3.1.0
+Jinja2==3.1.6
+Mako==1.3.9
+MarkupSafe==3.0.2
+PyJWT==2.10.1
+PySocks==1.7.1
+PyYAML==6.0.2
+SQLAlchemy-Utils==0.41.2
+SQLAlchemy==2.0.39
+Werkzeug==3.1.3
+alembic==1.15.1
+aniso8601==10.0.0
+attrs==25.3.0
+blinker==1.9.0
+cachelib==0.13.0
+certifi==2025.1.31
+cffi==1.17.1
+charset-normalizer==3.4.1
+click==8.1.8
+cryptography==44.0.2
+ecdsa==0.19.1
+filelock==3.18.0
+flake8-import-order==0.18.2
+flask-jwt-oidc==0.8.0
+flask-marshmallow==1.3.0
+flask-restx==1.3.0
+formsflow_api_utils @ git+https://github.com/aot-technologies/forms-flow-ai.git@a2c74f2c56fa1fabfda12fa6a044b81b30905950#subdirectory=forms-flow-api-utils
+fsspec==2025.3.0
+googleapis-common-protos==1.69.2
+grpcio==1.71.0
+gunicorn==23.0.0
+h11==0.14.0
+h2==4.2.0
+hpack==4.1.0
+huggingface-hub==0.29.3
+hyperframe==6.1.0
+idna==3.10
+importlib_metadata==8.6.1
+importlib_resources==6.5.2
+itsdangerous==2.2.0
+jsonschema-specifications==2024.10.1
+jsonschema==4.23.0
+kaitaistruct==0.10
+marshmallow-sqlalchemy==1.4.1
+marshmallow==3.26.1
 mpmath==1.3.0
-murmurhash==1.0.10
-networkx==3.1
-nltk==3.9.1
-numpy==2.2.3
-packaging==23.1
-plac==1.1.3
-preshed==3.0.9
-protobuf==5.29.1
-psycopg2-binary==2.9.7
+networkx==3.4.2
+numpy==2.2.4
+opentelemetry-api==1.31.0
+opentelemetry-exporter-otlp-proto-common==1.31.0
```

Comment:
I think the make setup command installs these packages, which are provided by api-utils